### PR TITLE
Make github disabled by default and enable passing options

### PR DESCRIPTION
**Pull request by skad0**
Resolves https://github.com/birhoff/md-to-bemjson/issues/23

### CODE REVIEW

**Comment by skad0** [comment]
@birhoff

File: lib/defaults.js [hunk 1]
```diff
@@ -1,5 +1,3 @@
 'use strict';
 
-module.exports = {
-    github: true
-};
+module.exports = {};
```
Comment (skad0) [inline lib/defaults.js]: Решил не удалять пока, как считаешь @birhoff ?

**Comment by birhoff** [comment]
@skad0 Please can you set commit message by conventional commit guide https://github.com/angular/angular/blob/master/CONTRIBUTING.md#type

I will generate release changelog of this message

File: README.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ API
 #### Options
 Parameter    | Type                 | Description
 -------------|----------------------|------------------------------
-`github`     | _boolean_            | Enables [github][github] support with [remark][remark] plugin [remark-github](https://github.com/wooorm/remark-github). Default — `true`.
+`github`     | _Object_, _boolean_            | Enables [github][github] support with [remark][remark] plugin [remark-github](https://github.com/wooorm/remark-github). Disabled by default.
```
Comment (birhoff) [inline README.md]: Default: `false`

File: lib/defaults.js (hunk 1 above)
Comment (birhoff) [inline lib/defaults.js]: yeah lets leave it

**Comment by birhoff** [comment]
Please change commit message and we merge this

**Comment by skad0** [comment]
>Please change commit message and we merge this

done

**Pull request merged**